R/payload_helpers.R

```diff
@@ -90,6 +91,12 @@ setup_payload_download_request <- function(version = NULL,
     payload <- c(payload, paste(
       '"pjnz": ', jsonlite::toJSON(pjnz, auto_unbox = TRUE, null = "null")))
   }
+  if (include_vmmc) {
+    path <- testthat::test_path("testdata", "vmmc.xlsx")
+    payload <- c(payload, paste(
+      '"vmmc":',
+      jsonlite::toJSON(file_object(path), auto_unbox = TRUE, null = "null")))
+  }
```

Comment:
Sorry, I'm a bit confused about which payload is getting downloaded here - it's some json that is included in the output zip? And here it's including some test data pending the real implementation being done by Jeff?

Reply:
Thanks, we chatted about this but just for history. This file is used to prepare a body for testing the API. It is in here instead of in test helpers because we also call this from the readme or the vignette I think. In some context outside of testing at least.